blog/2022-06-24-quickstart-kind/index.md

```diff
@@ -85,9 +84,9 @@ NOTES:
 1. Access the application URL by running these commands:
   Open http://console.paralus.local in browser.
 
-You can view the recovery link for admin user by running the following command once all the pods are running:
+You can view the default password for the admin user by running the following command once all the pods are running:
```

Comment:
do you want it to be like 

You can view the default password for the <admin-user> by running the following command once all the pods are running:

here <admin-user> will be replaced with the actual admin configured in helm e.g. admin@paralus.local by default